app/javascript/modCenter/singleArticle/__tests__/singleArticle.test.jsx

```diff
@@ -1,11 +1,10 @@
 /* eslint-disable jest/expect-expect */
 import { h } from 'preact';
 import { axe } from 'jest-axe';
-import { render, getNodeText, fireEvent } from '@testing-library/preact';
-
+import { render, getNodeText } from '@testing-library/preact';
 import SingleArticle from '../index';
 
-const testArticle1 = {
+const getTestArticle = () => ({
```

Comment:
Using a function to return an article instead of using an article directly so that it is not mutable across tests.

Reply:
👍